google-cloud-bigquery/lib/google/cloud/bigquery/load_job.rb

```diff
@@ -560,8 +560,13 @@ def time_partitioning_require_filter?
         ###
         # Checks if the destination table will be clustered.
         #
+        # See {LoadJob::Updater#clustering_fields=}, {Table#clustering_fields} and
+        # {Table#clustering_fields=}.
+        #
         # @see https://cloud.google.com/bigquery/docs/clustered-tables
-        #   Introduction to Clustered Tables
+        #   Introduction to clustered tables
+        # @see https://cloud.google.com/bigquery/docs/creating-clustered-tables
+        #   Creating and using clustered tables
         #
         # @return [Boolean, nil] `true` when the table will be clustered,
         #   or `false` otherwise.
```

Comment:
When is it nil, btw?

Reply:
That must be a mistake (it's untouched by this PR). I'll remove it.

Reply:
I found four such usages which appear to be a copy/paste mistake from other boolean methods that can return `nil`, for example:

```
        # @return [Boolean, nil] `true` when the table is range partitioned, or
        #   `false` otherwise, if the object is a resource (see {#resource?});
        #   `nil` if the object is a reference (see {#reference?}).
```